book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Great job so far. please kindly ensure A method for adding rentals should be present. the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. [see more](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)
eg
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```